context/version.ts

```diff
@@ -19,6 +26,25 @@ const dispatch: { [key: string]: version } = {
   "0.3.1": "0.3.1",
   "0.3.2": "0.3.2",
   "0.3.3": "0.3.3",
+  "0.3.4": "0.3.4",
+};
+
+// undefined means tzsafe does not support deploying that version
+export const CONTRACTS: { [k in version]: [string, any] | undefined } = {
+  "0.0.6": undefined,
+  "0.0.8": undefined,
+  "0.0.9": undefined,
+  "0.0.10": undefined,
+  "0.0.11": undefined,
+  //@ts-expect-error There was 2 versions of 0.0.11, so it means 2 codeHash.
+  "0.0.11b": undefined,
+  "0.1.1": [ct0_1_1, meta0_1_1],
+  "0.3.0": undefined,
+  "0.3.1": undefined,
+  "0.3.2": undefined,
+  "0.3.3": [ct0_3_3, meta0_3_3],
+  "0.3.4": [ct0_3_4, meta0_3_4],
```

Comment:
Is the tuple the contract code and the metadata?

Reply:
yes!

Reply:
Ok then can you change the type: 
`[contractCode: string, metadata: any]`